lib/execution_engine2/sdk/EE2Runjob.py

```diff
@@ -98,6 +99,7 @@ def _init_job_rec(
         {_SERVICE_VER} (app version)
         {_APP_ID} (app UI)
         {_SOURCE_WS_OBJECTS} (collected workspace objects for this app)
+        {_BATCH_ID} (parent of the job for EE2 batch jobs, the parent should be updated)
```

Comment:
I think the terminology is now "Coordinating" or "Container" or something instead of parent

Reply:
Yep, this was before the parent purge happened, and it got missed as well